packages/boot/src/__tests__/unit/mixins/boot.mixin.unit.ts

```diff
@@ -36,6 +36,14 @@ describe('BootMxiin unit tests', () => {
     expect(booterBinding.tagMap).to.containEql({artifactType: 'xsd'});
   });
 
+  it('binds booter with `@bind` using a custom binding key', async () => {
+    app.booters(TestBooterWithCustomBindingKey);
+    const booterBinding = app.getBinding(
+      `io.loopback.custom.binding.TestBooterWithNamedBind`,
+    );
+    expect(booterBinding.tagMap).to.containEql({artifactType: 'bmp'});
```

Comment:
you may want to test the `configured` property equals to `true` as well

Reply:
I had to update the test cases to use it's own `app` instance and invoke `app.boot()`, but done :)